api_app/analyzers_manager/views.py

```diff
@@ -0,0 +1,9 @@
+# from django.shortcuts import render
+# from rest_framework import serializers as rfs
+from rest_framework import viewsets
+
+from api_app.analyzers_manager.serializers import AnalyzerSerializer
+
+
+class AnalyzerViewSet(viewsets.ModelViewSet):
```

Comment:
use `viewsets.ReadOnlyModelViewSet` because we do not wish to allow update/create at the moment.